main.css

```diff
@@ -0,0 +1,79 @@
+/* Default Styles */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+/* Main Styles */
+
+body {
+  font-family: "Poppins", sans-serif;
+  font-size: 16px;
+  background: #e5e5e5;
+  padding: 10px 25px;
```

Comment:
- [ ] **[REQUIRED CHANGES]** As we can see in the provided figma template, the background for this section should be white. Can you fix it for us, please?
- [ ] **[REQUIRED CHANGES]** If we take a look at your page, we will be able to see that part of the image is cutting right next to the borders of the screen. This is due to you setting your padding directly on the body. If you remove your padding from the body and change your padding as I proposed below, you will notice that this gap between the borders and the image will no longer exist. You might also need to add similar padding to the toolbar as well.

This is the gap that I'm talking about:
<img src="https://user-images.githubusercontent.com/47700024/138102867-32b31240-40fa-4fa4-a89e-ec3cdf74c921.png" width="200px">

Reply:
From the video on how to use the Figma files given in the project's Sneak peek: your portfolio, We are told to use the grouped section design as an image and export it from the Figma file to be used as a background image for the headline section. If I set the section to have a white background image, then the beautiful design that is used will be lost from the looks.